spec/features/admin/index_spec.rb

```diff
@@ -126,4 +126,50 @@
+    item_7 = walmart.items.create!(name: "wine glass", description: "stemless", unit_price: 350)
+    item_8 = walmart.items.create!(name: "banjo", description: "five string", unit_price: 30100)
+    item_9 = walmart.items.create!(name: "golf tees", description: "2 1/2 inch", unit_price: 100)
+    item_10 = walmart.items.create!(name: "radio", description: "AM/FM", unit_price: 900)
+    item_11 = walmart.items.create!(name: "adult diaper", description: "family size", unit_price: 400)
+    invoice_1 = bob.invoices.create!(status: 1, created_at: '05 Apr 2022 00:53:36 UTC +00:00')
+    invoice_2 = bob.invoices.create!(status: 1, created_at: '04 Apr 2022 00:53:36 UTC +00:00')
+    invoice_3 = bob.invoices.create!(status: 1, created_at: '03 Apr 2022 00:53:36 UTC +00:00')
+    invoice_4 = bob.invoices.create!(status: 1, created_at: '02 Apr 2022 00:53:36 UTC +00:00')
+    invoice_5 = bob.invoices.create!(status: 1, created_at: '01 Apr 2022 00:53:36 UTC +00:00')
+    invoice_6 = bob.invoices.create!(status: 1, created_at: '06 Apr 2022 00:53:36 UTC +00:00')
+    invoice_7 = bob.invoices.create!(status: 1, created_at: '07 Apr 2022 00:53:36 UTC +00:00')
+    invoice_8 = bob.invoices.create!(status: 1, created_at: '08 Apr 2022 00:53:36 UTC +00:00')
+    invoice_9 = bob.invoices.create!(status: 1, created_at: '09 Apr 2022 00:53:36 UTC +00:00')
+    invoice_10 = bob.invoices.create!(status: 1, created_at: '10 Apr 2022 00:53:36 UTC +00:00')
+    invoice_11 = bob.invoices.create!(status: 1, created_at: '11 Apr 2022 00:53:36 UTC +00:00')
+    InvoiceItem.create!(invoice_id: invoice_1.id, item_id: item_1.id, quantity: 1, status: 1)
+    InvoiceItem.create!(invoice_id: invoice_2.id, item_id: item_2.id, quantity: 1, status: 2)
+    InvoiceItem.create!(invoice_id: invoice_3.id, item_id: item_3.id, quantity: 1, status: 2)
+    InvoiceItem.create!(invoice_id: invoice_4.id, item_id: item_4.id, quantity: 1, status: 0)
+    InvoiceItem.create!(invoice_id: invoice_5.id, item_id: item_5.id, quantity: 1, status: 2)
+    InvoiceItem.create!(invoice_id: invoice_6.id, item_id: item_6.id, quantity: 1, status: 2)
+    InvoiceItem.create!(invoice_id: invoice_7.id, item_id: item_7.id, quantity: 1, status: 1)
+    InvoiceItem.create!(invoice_id: invoice_8.id, item_id: item_8.id, quantity: 1, status: 0)
+    InvoiceItem.create!(invoice_id: invoice_9.id, item_id: item_9.id, quantity: 1, status: 2)
+    InvoiceItem.create!(invoice_id: invoice_10.id, item_id: item_10.id, quantity: 1, status: 2)
+    InvoiceItem.create!(invoice_id: invoice_11.id, item_id: item_11.id, quantity: 1, status: 2)
+
+    visit '/admin'
+    
+    expect(invoice_11.id).to appear_before(invoice_10.id)
+    expect(invoice_10.id).to appear_before(invoice_9.id)
+    expect(invoice_1.id).to appear_before(invoice_2.id)
+    expect(invoice_2.id).to appear_before(invoice_3.id)
+    expect(invoice_3.id).to appear_before(invoice_4.id)
+    expect(page).to have_content('Tuesday, April 05, 2022')
```

Comment:
can we had another merchant and store to not appear so we can test that its only displaying the results we want? maybe we add more testing on next review